Content.Server/Administration/Commands/WarpCommand.cs

```diff
@@ -118,8 +120,7 @@ public void Execute(IConsoleShell shell, string argStr, string[] args)
                 }
 
                 var xform = _entManager.GetComponent<TransformComponent>(playerEntity);
-                var xformSystem = _entManager.System<SharedTransformSystem>();
-                xform.Coordinates = coords;
+                xformSystem.SetCoordinates(playerEntity, coords);
                 xformSystem.AttachToGridOrMap(playerEntity, xform);
```

Comment:
```suggestion
                xformSystem.AttachToGridOrMap(playerEntity);
```
no need for the `GetComponent` above since the function already resolves it using a query if null

Reply:
Good point!